kubernetes/README.md

```diff
@@ -71,10 +72,77 @@ These are pause containers (docker_image:gcr.io/google_containers/pause.*) that
 
 The docker_daemon check ignores them through a default exclusion list, but they will show up for K8s metrics like *kubernetes.cpu.usage.total* and *kubernetes.filesystem.usage*.
 
+### How to elect a leader agent in order to avoid duplicated events?
+
+Since dd-agent 5.17, the agent has a leader election option for kubernetes.
+Every agent is a leader candidate, and they will perform a leader election among themselves using kubernetes components.
```

Comment:
"Every member of the Datadog DaemonSet is considered a leader candidate, and pods will perform a leader election among themselves through Kubernetes."

Can we speak to which api or method kubernetes provides for this?